docs/graphql/graphql-spans.md

```diff
@@ -9,10 +9,9 @@ linkTitle: GraphQL Server
 This document defines semantic conventions to apply when instrumenting the GraphQL implementation. They map GraphQL
 operations to attributes on a Span.
 
-The **span name** MUST be of the format `<graphql.operation.type> <graphql.operation.name>` provided that
-`graphql.operation.type` and `graphql.operation.name` are available. If `graphql.operation.name` is not available, the
-span SHOULD be named `<graphql.operation.type>`. When `<graphql.operation.type>` is not available, `GraphQL Operation`
-MAY be used as span name.
+The **span name** MUST be of the format `<graphql.operation.type>` provided that
```

Comment:
Most other references I could find for `**span name**` use `SHOULD`-level requirements rather than `MUST`-level requirements. I left this as `MUST` to remain closer to the previous convention, but I can also see benefits in making it a `SHOULD`-level requirement.

Reply:
I'd support SHOULD, especially because the next sentence implies that operation type is not always available.

Reply:
Excellent. Updated in 3806796